package.json

```diff
@@ -18,18 +18,11 @@
     "postpublish": "git push origin && git push origin --tags",
     "build": "rimraf dist && tsc -p tsconfig.json && tsc -p tsconfig.esm.json",
     "pretest": "yarn build",
-    "test": "tape dist/cjs/test/{unit,compat}/*.js",
+    "test": "tape dist-test/cjs/test/{unit,compat}/*.js",
     "test:browser": "airtap ./test/{unit,compat}/*.ts",
-    "bsamples": "cd ../tools/bsamples && cargo run",
-    "lint": "prettier -c ./src/",
-    "lint:fix": "prettier --write ./src",
-    "tsc": "tsc --noEmit",
-    "doc": "rimraf ../docs/beet && typedoc",
-    "doc:update": "./sh/update-docs",
-    "doctoc": "doctoc README.md"
```

Comment:
Thanks, I wasn't sure about:
- `bsamples` which seemed like we could want to use it (it's for testing, right?)
- `doc*` because we could want to use it some day
- `lint*` was probably ok to leave. Thanks for removing `.prettierrc` though!